test/integration/job/job_test.go

```diff
@@ -1820,6 +1820,137 @@ func TestJobPodReplacementPolicy(t *testing.T) {
 	}
 }
 
+func TestJobPodReplacementPolicyOnRestart(t *testing.T) {
+	const podCount int32 = 2
+	nonIndexedCompletion := batchv1.NonIndexedCompletion
+	var podReplacementPolicy = func(obj batchv1.PodReplacementPolicy) *batchv1.PodReplacementPolicy {
+		return &obj
+	}
+	cases := map[string]struct {
+		podReplacementPolicyEnabled bool
```

Comment:
Does not seem needed if all the scenarios follow the path: enabled -> disabled -> enabled.

Reply:
I am stuck on this actyall.  It looks like the disable (last step) doesn’t work.  I can’t really understand why.  I can disable and enable without issue but that third test is failing.  It’s like the toggling of the feature doesn’t work.